src/applications/disability-benefits/686/helpers.jsx

```diff
@@ -105,7 +105,7 @@ export const otherRelationshipDescription = (
 );
 
 export function isVeteran(item) {
```

Comment:
Should we change the name of this function? It's now if you're not a veteran, not that you are a veteran.

Reply:
k